bears/c_languages/codeclone_detection/ClangComplexityBear.py

```diff
@@ -0,0 +1,98 @@
+from clang.cindex import Index, CursorKind
+
+from coalib.bears.GlobalBear import GlobalBear
+from bears.c_languages.ClangBear import clang_available
+from coalib.results.Result import Result
+
+
+class ClangComplexityBear(GlobalBear):
+    """
+    Calculates cyclomatic complexity of each function and displays it to the
+    user.
+    """
+    check_prerequisites = classmethod(clang_available)
+    decisive_cursor_kinds = {
+        CursorKind.IF_STMT, CursorKind.WHILE_STMT, CursorKind.FOR_STMT
```

Comment:
I think he meant you should add the cursor kind for switch-case (because `if else` and `switch case` are similar)
Is that what you're talking about ?